### PR TITLE
Adding text display. Press 'z' to show player name

### PR DESCRIPTION
**Pull request by jsnprtr**
#### What's this PR do?
Adds name display to game area on button press 'z'

##### Background context
Thought it would be cool.

#### Where should the reviewer start?
app/lib/player.rb - not sure if this is the correct way to do this

#### How should this be manually tested?
Press 'z'

### CODE REVIEW

File: app/lib/player.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ module Hammeroids
   # Represents player stores player UUID in redis.
   class Player
     attr_accessor :id
+    attr_accessor :name
```
Comment (Dannyguk) [inline app/lib/player.rb]: you can just do 
```attr_accessor :id, :name```

https://apidock.com/ruby/Module/attr_accessor

File: app/javascript/src/game/view.js [hunk 2]
```diff
@@ -15,15 +15,17 @@ export class View {
     this._context.fillRect(0, 0, this._gameWidth, this._gameHeight);
   };
 
-  drawObject = (object) => {
-    object.getDrawer().draw(this._context);
-  };
+  // drawObject = (object) => {
```
Comment (Dannyguk) [inline app/javascript/src/game/view.js]: Remove commented out stuff?

File: app/lib/player.rb (hunk 1 above)
Comment (jsnprtr) [reply app/lib/player.rb]: Nice

File: app/javascript/src/game/view.js (hunk 2 above)
Comment (jsnprtr) [reply app/javascript/src/game/view.js]: Yup

**Pull request merged**